src/components/ColorPicker/ColorPicker.vue

```diff
@@ -31,11 +31,44 @@ actual pickers:
 - One more advanced picker that provides the full color spectrum;
 
 ### Usage
+You can pass in an HTML element in the default slot, it will be used as the 
+trigger. You can also bind the property `open` to your variables.
 
-```
-<ColorPicker v-model="color" />
-<ColorPicker :value.sync="color" />
-<ColorPicker :value="color" @change="updateColor" />
+```vue
+<template>
+	<!-- using v-model and passing in a trigger -->
+  	<ColorPicker v-model="color">
+		<button> Trigger </button>
+	<ColorPicker />
+	<!-- using `:value.sync` and bindingto the open state to the oepen prop -->
+	<ColorPicker :value.sync="color" :open="open" />
```

Comment:
Having it opened by default doesn't make sense in the docs as it's confusing and people are most likely to click somewhere else on the page trigering the click-outside directive and closing it :)

I would go for 2 examples only, one with a `<button>Trigger</button>` inside as the trigger, and another with the `<button @click="open = !open">Trigger</button>` outside controlling the open value

Reply:
open should be false by default